### ISSUE TITLE
DOM panel options menu

### ISSUE THREAD

**Issue by janodvarko** [firebug/firebug.next#154]
DOM panel options menu is missing.

![dom-options](https://cloud.githubusercontent.com/assets/37785/4736013/efd28b26-59e7-11e4-9ec2-2ea3f04362d7.png)

Honza

**Comment by SebastianZ** [comment]
> I filled a bug about the stats. Sounds like we should finally collect the data.
> https://github.com/firebug/firebug.next/issues/13

As telemetry is already in place, it would be interesting to know what the statistics say.

Note that there was a [request for this on Stack Overflow](http://stackoverflow.com/q/28465134/432681) yesterday.

Sebastian

**Comment by janodvarko** [comment]
Firebug is already collecting telemetry measures, data are uploaded, but I still can't access and mine it from the (online) database :-(

Honza

**Comment by fflorent** [comment]
> Right. One would probably have to use a separate dropdown menu (still much more discoverable than the options menu, because they serve different purposes), or possibly icons. 

Yeah, the same as for the drop-down list next to the Console filters.

> Firebug is already collecting telemetry measures, data are uploaded, but I still can't access and mine it from the (online) database :-(

Pfff.... Why is that so complicated? :(

Florent

**Comment by janodvarko** [comment]
> Pfff.... Why is that so complicated? :(

I don't know, it's somehow hard to reach the right people and get the right help....

Honza
